data/tribes/scripting/help/building_help.lua

```diff
@@ -889,41 +821,85 @@ end
 function building_help_production_section(tribe, building_description)
    -- Produced items
    local result = h2(_("Production"))
-   if (building_description.output_ware_types[1] or building_description.output_worker_types[1]) then
-      local checked_programs ={}
-      for i, ware_description in ipairs(building_description.output_ware_types) do
-         programs, ware_counters, ware_strings = programs_wares_count(tribe, building_description, ware_description)
-         -- check if the ware is collected (no producing program)
-         if #programs == 0 then
-            result = result .. h3(_("Ware produced:"))
-            result = result .. help_ware_amount_line(ware_description, 1)
-            break
-         end
+   local checked_programs = {}
+   if (building_description.type_name == "trainingsite") then
+      programs, ware_counters, ware_strings = programs_wares_count(tribe, building_description, nil)
+      if (not checked_programs[program]) then
          for j, program in ipairs(programs) do
-            if (ware_counters[program] > 0) and not checked_programs[program] then
+            trained = building_description:trained_soldiers(program)
+            -- TRANSLATORS: Building Encyclopedia Skill trained by a trainingsite
+            result = result .. h3(_("Trained skill:"))
```

Comment:
How about:

```suggestion
            result = result .. h3(_("Training step:"))
```

Reply:
for me the first focus is on the property or skill. only afterwards on the steps. therefore I'd rather stay with skill here.

Reply:
I see. But "Trained skill" doesn't feel natural English to me (even though the direct translation is perfectly fine in hungarian), but of course I'm not a native speaker either.

Maybe @andy5995 or @RHolton could help us here.

Reply:
I'm not sure yet what's intended. It looks like you might want the equivalent of "Promotion" or "Promoted" level. That term is used in MegaGlest and 0ad when a unit stays the same, (doesn't morph) but receives an armor or hp increase.

Reply:
@andy5995 indeed the headline should denote that a soldier gets a promotion / a rise in level, but I wanted to emphasize on the exact capapbility of our 4 military capabilities that gets promoted. I do not know a better word then skill for it. 
As @tothxa is unsure whether "Trained skill" is the proper way to say this in english maybe it is just a matter of sequence. Would "Skill trained:" sound better?

Reply:
@hessenfarmer "Skill trained" doesn't sound better to me. how about "Skill-step" or "skill-boost" or "skill-booster"? I'll give it some more thought...

Reply:
well this is the help of the building, therefore I wanted to have a short form for "this building trains soldier in [skill]". With my suggestion I just took the other short headlinbes into account which are "Wares produced:" and "Wares consumed:". I will have a look whether and how it would be feasible to include the Skill already in the Headline, resulting in "Health Improvement", " Attack Training" "Defense Training" and "Evade Training:".    
could this be a solution?